sound/soc/sof/ipc4-topology.c

```diff
@@ -1054,6 +1055,41 @@ sof_ipc4_prepare_copier_module(struct snd_sof_widget *swidget,
 	if (ret < 0)
 		return ret;
 
+	switch (swidget->id) {
+	case snd_soc_dapm_dai_in:
+	case snd_soc_dapm_dai_out:
+	{
+		switch (ipc4_copier->dai_type) {
+		case SOF_DAI_INTEL_ALH:
+		{
+			struct sof_ipc4_alh_configuration_blob *blob;
+			u32 ch_map;
+			int i;
+
+			blob = kzalloc(sizeof(*blob), GFP_KERNEL);
+			if (!blob)
+				return -ENOMEM;
+
+			blob->alh_cfg.count = 1;
+			blob->alh_cfg.mapping[0].alh_id = copier_data->gtw_cfg.node_id;
+			blob->gw_attr.lp_buffer_alloc = 0;
+			copier_data->gtw_cfg.config_length = sizeof(*blob) >> 2;
+
+			/* Get channel_mask from ch_map */
+			ch_map = copier_data->base_config.audio_fmt.ch_map;
+			for (i = 0; ch_map > 0; i++) {
+				if ((ch_map & 0xf) != 0xf)
+					 blob->alh_cfg.mapping[0].channel_mask |= BIT(i);
+				ch_map >>= 4;
+			}
+
+			ipc4_copier->copier_config = (uint32_t *)blob;
+			break;
+		}
+		}
+	}
+	}
```

Comment:
indentation issues here?

Reply:
just an extra curly braces outside the inner switch/case